packages/liveblocks-core/src/room.ts

```diff
@@ -3037,7 +3037,9 @@ export function createRoom<
 
   function yjsStatusDidChange(status: YjsSyncStatus) {
     return syncSourceForYjs.setSyncStatus(
-      status === "synchronizing" ? "synchronizing" : "synchronized"
+      status === "synchronizing" || status === "loading"
+        ? "synchronizing"
+        : "synchronized"
```

Comment:
This is another bug fix - sync status for Y.js is either `loading`, `synchronizing` or `synchronized`. According to the previous logic, we were displaying `synchronized` for `loading` states.

Reply:
this was an intentional decision, but I can't remember exactly why. There was a lot of discussion about it. I'll see if I can dig up that conversation.